main.go

```diff
@@ -16,6 +16,78 @@
 
 package main
 
+import (
+	"cmd-exclude-prefixes-k8s/internal/prefixcollector"
+	"cmd-exclude-prefixes-k8s/internal/utils"
+	"context"
+	"sync"
+
+	"github.com/networkservicemesh/sdk/pkg/networkservice/common/excludedprefixes"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/jaeger"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
+)
+
+// Config - configuration for cmd-exclude-prefixes-k8s
+type Config struct {
+	ExcludedPrefixes   []string `desc:"List of excluded prefixes" split_words:"true"`
+	ConfigMapNamespace string   `default:"default" desc:"Namespace of kubernetes config map" split_words:"true"`
+}
+
+const envPrefix = "nsm"
```

Comment:
I guess we need to set here the name of the application

Reply:
Fixed.